tests/integration/controller/kyma/manifest_test.go

```diff
@@ -329,6 +330,42 @@ var _ = Describe("Test Reconciliation Skip label for Manifest", Ordered, func()
 	})
 })
 
+var _ = Describe("Modules can only be referenced via official label", Ordered, func() {
```

Comment:
I would say rename `official label` to `module name` since `official label` is not clear enough